styles.css

```diff
@@ -0,0 +1,100 @@
+* {
+  font-family: Poppins, sans-serif;
+  padding: 0;
+  margin: 0;
+}
+
+.main-block {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  min-width: 375px;
+}
+
+.head-menu {
+  display: flex;
```

Comment:
### OPTIONAL
- [ ] For DRY CSS you should combine duplicate properties. Example, you can create class `.d-flex { display: flex}` and use this class in HTML file.Flex is just an example you can do it to duplicate properties.



- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and css best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)